src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * deals with loading and saving tasks in the file
+ */
+public class Storage {
+    private String directory;
+    private String filePath;
+
+    public Storage(String directory, String filePath) {
+        this.directory = directory;
+        this.filePath = filePath;
+    }
+
+    /**
+     * rewrite the file with newly edited tasks after the user quits the app
+     * @param list_of_inputs
+     * @throws IOException
+     */
+    public void reSavingFiles (ArrayList<Task> list_of_inputs) throws IOException {
+        //start saving the results for next return
+        for (int i = 0; i < list_of_inputs.size(); i++) {
+            if (i == 0) {
+                writeToFile(list_of_inputs.get(i).message(),filePath);
+            } else {
+                list_of_inputs.get(i).updateData(filePath);
+
+            }
+        }
+    }
+
+    /**
+     * read data from file
+     * @param filePath
+     */
```

Comment:
For Javadoc method header comments, maybe you can consider the following format as stated in the coding standard:
```java
/**
 * Reads data from file.
 *
 * @param filePath the file path to read the data from.
 */
```
I noticed similar issues (missing description, no starting verb, starting verb is non-compliant, no link break between description and `@param`, no description for `@param`, `@return`) in several other places too.